runtime/modules/inccommand.lua

```diff
@@ -38,17 +38,19 @@ local function refresh_completion()
   then
     return
   end
-  local items = get_items()
-  if items then
-    vim.rpcnotify(vim.g.vscode_channel, "redraw", { "wildmenu_show", { items } })
+  local lines = get_lines()
+  if lines then
+    local items = {}
+    for _, line in ipairs(lines) do
+      table.insert(items, { line, "", "", "" })
+    end
+    vim.rpcnotify(vim.g.vscode_channel, "redraw", { "popupmenu_show", { items, -1, 0, 0, -1 } })
```

Comment:
Interesting. Are we "faking" a Nvim redraw event?

Reply:
Yeah. This isn't a core feature, but just a lua gimmick which feeds the inccommand preview buffer into the cmdline suggestions. It's self-contained so not a maintenance burden/complexity add.